samples/WinRTSample/WinRTSample.vsimporter/WinRTSample-WinStore10/WinRTSample.vcxproj

```diff
@@ -201,5 +215,7 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
+  <ItemGroup>
+    <None Include="project.json" />
+  </ItemGroup>
```

Comment:
WinRTSample and KVO Perf didn't have the original xcode around so I manually did them.

---
In reply to: [102804684](https://github.com/Microsoft/WinObjC/pull/2056#discussion_r102804684) [](ancestors = 102804684)